### ISSUE TITLE
Default user/password

### ISSUE THREAD

**Issue by kb2ear** [openmhz/trunk-server#10]
The default username and password do not work on the account.openmhz.test site.

**Comment by natecarlson** [comment]
Yeah, I noticed this too. I just created an account via the register page with no issues.

**Comment by sally-yachts** [comment]
I've got a test environment stood up that appears to be healthy, but I don't see a way to create any admin credentials? I can register a normal user through the UI, and that user can login to admin.site.com. However, that account can only see the Radio Systems section. When trying to add a system it throws  {"success":false,"message":{}} in the response and Object { type: "CREATE_ERROR_SYSTEM", data: undefined } in the console, I'm assuming because it's a regular user without admin credentials.
